src/Core/src/Primitives/GridLength.cs

```diff
@@ -9,22 +9,16 @@ namespace Microsoft.Maui
 	public struct GridLength
```

Comment:
While we are in here making readonly changes, can we just make the whole struct readonly? That is our preferred approach in dotnet/runtime.

```suggestion
	public readonly struct GridLength
```